### DISCUSSION TITLE
Custom rulesets should not appear in online overlays

### DISCUSSION THREAD

**Discussion by EVAST9919** [ppy/osu#22155] [Ideas] [answered]
![Снимок экрана (307)](https://user-images.githubusercontent.com/22874522/212059648-d8635b01-d721-4840-9668-d2f189d26261.png)

`RulesetSelector` probably should have `bool addCustomRulesets` in the ctor or something.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/22158
